src/Dashboard.js

```diff
@@ -11,7 +12,8 @@ export default class Dashboard extends React.Component {
     super(props);
     ReactModal.setAppElement("body");
     this.state = {
-      showModal: false
+      showModal: false,
+      expandSidebar: false
```

Comment:
I learned things about lifting state! v glad the react docs were understandable for this